src/components/Calculator.js

```diff
@@ -0,0 +1,48 @@
+import './calculator.css';
+import PropTypes from 'prop-types';
+
+function Calculator() {
+  return (
+    <div className="calculator">
+      <div className="console">0</div>
+      <div className="btn-group">
+        <Button content="ac" />
+        <Button content={'\u00B1'} />
+        <Button content={'\u0025'} />
+        <Button content={'\u00f7'} />
+        <Button content="7" />
+        <Button content="8" />
+        <Button content="9" />
+        <Button content={'\u2715'} />
+        <Button content="4" />
+        <Button content="5" />
+        <Button content="6" />
+        <Button content={'\u2212'} />
+        <Button content="1" />
+        <Button content="2" />
+        <Button content="3" />
+        <Button content={'\u002b'} />
+        <Button content="0" className="zero" />
+        <Button content={'\u22c5'} />
+        <Button content={'\u003d'} className="equal" />
+      </div>
+    </div>
+  );
+}
+
+function Button({ content, className }) {
+  return (
+    <button type="button" className={className}>{content}</button>
+  );
+}
+
```

Comment:
- [ ] You did a great job of creating the `Button` component but I highly recommend you move this `Button` component to its own file in order to keep your repo clean and professional. Make sure you import this component where it needs to be used.

Reply:
well resolved! ✔️